README.md

```diff
@@ -1,7 +1,17 @@
-# remix
+# REMIX
 Ethereum IDE and tools for the web
 
-INSTALLATION:
+## REMIX WEBSITE:
+
+Remix is avalaible at http://ethereum.github.io/remix. Do not use https if connecting to web3 is required (should be always like that with remix).
```

Comment:
You can also use it inside mist - then you *should* use https, but mist does not work currently.